src/guide/4-initial-conditions/2-initial-properties.rst

```diff
@@ -4,7 +4,7 @@ Initialising Agent Variables
 Initialising a Population
 -------------------------
 
-To initialise a population, iterate over the ``AgentVector`` object and set the properties for each agent:
+To initialise a population, iterate over the ``AgentVector`` object and set the variables for each agent:
 
 ** TODO: not very idiomatic - python should use enumerate I think, and range based for for cpp**
```

Comment:
Yeah treating as iterable does work, it's just awkward if manually assigning IDs. Given we now have manual IDs you could change the example to something else.

C style random is pretty grim here, but rough example.

```python
for instance in population:
    instance.setVariableFloat("foo", numpy.random.normal());
```

```c++
for (auto instance : population) {
    instance.setVariable<float>("foo", rand()/(float)RAND_MAX);
}
```